docs/site/package.json

```diff
@@ -13,21 +13,21 @@
     "clean": "rimraf build"
   },
   "devDependencies": {
-    "babel-cli": "^6.18.0",
+    "babel-cli": "^6.22.2",
```

Comment:
I haven't looked in to Yarn, so I didn't realise it's caching doesn't invalidate when a package is updated. That doesn't sound very smart. No wonder it's so fast! 😄